pages/index.html

```diff
@@ -24,8 +24,8 @@ <h3>SSL Black and White Hulls Challenge</h3>
 
 <h3>SSL Teams vs Teams Passing Challenge</h3>
```

Comment:
Good catch with the challenge name. Can you also adapt it for the title for consistency? :)

Reply:
Done!